placeholder value with space: `panic: runtime error: index out of range`

System info:

- macOS 10.12.6
- sman installed via `bash -c "$(curl https://raw.githubusercontent.com/tokozedg/sman/master/install.sh)"`

```bash
# s run -f shell sed:replace 'a d' b c
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/tokozedg/sman/lib.(*Snippet).SetInputs(0xc8200efbb8, 0xc82006ab00, 0x4, 0x6)
	/Users/Tornike/gocode/src/github.com/tokozedg/sman/lib/snippet.go:36 +0x112
github.com/tokozedg/sman/lib.run(0x7fff5fbff5f2, 0xb, 0xc82006ab00, 0x4, 0x6)
	/Users/Tornike/gocode/src/github.com/tokozedg/sman/lib/run.go:87 +0x29f
github.com/tokozedg/sman/lib.glob.func2(0x50ace0, 0xc82006aaf0, 0x5, 0x7)
	/Users/Tornike/gocode/src/github.com/tokozedg/sman/lib/run.go:135 +0x178
github.com/spf13/cobra.(*Command).execute(0x50ace0, 0xc82006aa80, 0x7, 0x7, 0x0, 0x0)
	/Users/Tornike/gocode/src/github.com/spf13/cobra/command.go:569 +0x869
github.com/spf13/cobra.(*Command).ExecuteC(0x50aae0, 0x50ace0, 0x0, 0x0)
	/Users/Tornike/gocode/src/github.com/spf13/cobra/command.go:656 +0x56b
github.com/spf13/cobra.(*Command).Execute(0x50aae0, 0x0, 0x0)
	/Users/Tornike/gocode/src/github.com/spf13/cobra/command.go:615 +0x2d
main.main()
	/Users/Tornike/gocode/src/github.com/tokozedg/sman/main.go:8 +0x23

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	/usr/local/Cellar/go/1.5.3/libexec/src/runtime/asm_amd64.s:1721 +0x1
```

Notes:

- command runs fine if there's no space in value

--- a/sman.rc
+++ b/sman.rc
@@ -2,9 +2,9 @@
 
 s() {
   if [[ ${1} == "run" ]] || [[ ${1} == "r" ]]; then
-    eval "$(sman $@)"
+    eval "$(sman "$@")"
   else
-    sman $@
+    sman "$@"
   fi
 }
 

